content/docs/fairing/gcp-local-notebook.md

```diff
@@ -0,0 +1,230 @@
+    ```
+
+## Install and configure the Google Cloud SDK
+
+In order to use Kubeflow Fairing to train or deploy to Kubeflow on GKE,
+or Cloud Machine Learning Engine, you must configure
+your development environment with access to GCP. 
+
+1.  If you do not have the Cloud SDK installed, [install the
+    Cloud SDK][gcloud-install].
+
+1.  Use `gcloud` to set a default project.
+
+    ```bash
+    export PROJECT_ID=<your-project-id>
+    gcloud config set project $PROJECT_ID
+    ```
+
+1.  Kubeflow Fairing needs a service account to make API calls to GCP. The
+    recommended way to provide Fairing with access to this
+    service account is to set the `GOOGLE_APPLICATION_CREDENTIALS` environmental
+    variable. To check for the `GOOGLE_APPLICATION_CREDENTIALS` environmental
+    variable, run the following command:
+
+    ```bash
+    ls "$GOOGLE_APPLICATION_CREDENTIALS"
+    ```
+
+    The response should be something like this:
+
+    ```bash
+    /.../.../key.json
+    ```
+
+    If you do not have a service account, then create one and grant it
+    access to the required roles.
+
+    ```bash
+    export SA_NAME=<your-sa-name>
+    gcloud iam service-accounts create $SA_NAME
+    gcloud projects add-iam-policy-binding $PROJECT_ID --member serviceAccount:$SA_NAME@$PROJECT_ID.iam.gserviceaccount.com  --role 'roles/editor'
+    ```
+
+    Create a key for your service account.
+
+    ```bash
+    gcloud iam service-accounts keys create ~/key.json --iam-account $SA_NAME@$PROJECT_ID.iam.gserviceaccount.com
+    ```
+
+    Create the `GOOGLE_APPLICATION_CREDENTIALS` environmental variable.
+
+    ```bash
+    export GOOGLE_APPLICATION_CREDENTIALS=~/key.json
+    ```
+
+## Set up Docker
+
+You need to have Docker installed to use Kubeflow Fairing. Fairing packages
+your code as a Docker image and executes it in the remote cluster. To check
+if your local Docker daemon is running, run the following command:
+
+```bash
+docker ps
+```
+
+*  If you get a message like `docker: command not found`, then [install
+   Docker](https://docs.docker.com/install/).
+*  If you get an error like `Error response from daemon: Bad response from
+   Docker engine`, then [restart your docker daemon][docker-start].
+*  If you are using Linux and you use sudo to access Docker, follow these
+   steps to [add your user to the `docker` group][docker-non-root]. Note, the
+   `docker` group grants privileges equivalent to the root user. To learn more
+   about how this affects security in your system, see the guide to the
+   [Docker daemon attack surface][docker-attack].
+
+Authorize Docker to access your [GCP Container Registry][container-registry]. 
+
+```bash
+gcloud auth configure-docker
+```
+
+## Set up Kubeflow
+
+Use the following instructions to set up and configure your Kubeflow and
+development environments for training and prediction from Kubeflow Fairing.
+
+1.  You need to have kubectl installed to use Kubeflow Fairing. To set up
```

Comment:
>You need to have kubectl installed to use Kubeflow Fairing.

We don't for GCP installations. We use gcloud to get the k8s creds.

Reply:
Done.

Reply:
I also revised the wording on the "update kubeconfig" step to remove the reference to kubectl.